waspc/data/Generator/templates/server/src/_types/index.ts

```diff
@@ -83,5 +83,7 @@ type ContextWithUser<Entities extends _Entity[]> = Expand<Context<Entities> & {
 // password field from the object there, we must do the same here). Ideally,
 // these two things would live in the same place:
 // https://github.com/wasp-lang/wasp/issues/965
-export type SanitizedUser = Omit<{= userEntityName =}, 'password'>
+export type SanitizedUser = {= userEntityName =} & {
```

Comment:
We are not only providing the `User` (business logic user) but also the `Auth` (auth user)